### PR TITLE
2021-12-25-xlm_roberta_large_token_classifier_conll03_de

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ner_converter = NerConverter()\
+      .setInputCols(["sentence", "token", "ner"])\
+      .setOutputCol("ner_chunk")
+      
+nlpPipeline = Pipeline(stages=[documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter])
+
+empty_data = spark.createDataFrame([[""]]).toDF("text")
+
+model = nlpPipeline.fit(empty_data)
+text = """Ich bin Michael, ich lebe in Stuttgart und habe im Mercedes-Benz Werk gearbeitet."""
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md:63]: Can we pick a real German sentence from Wikipedia?

File: docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  .setInputCols(Array("sentence","token"))\
+  .setOutputCol("ner")
+
+ner_converter = NerConverter()\
+      .setInputCols(Array("sentence", "token", "ner"))\
+      .setOutputCol("ner_chunk")
+      
+val pipeline = new Pipeline().setStages(Array(documentAssembler, sentenceDetector, tokenizer, tokenClassifier, ner_converter))
+
+val example = Seq.empty["Ich bin Michael, ich lebe in Stuttgart und habe im Mercedes-Benz Werk gearbeitet."].toDS.toDF("text")
```
Comment (josejuanmartinez) [inline docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md:89]: Can we pick a real German sentence from Wikipedia?

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Can you take a look at the requested changes? Thank you.

File: docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md (hunk 2 above)
Comment (murat-gunay) [reply docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md]: @josejuanmartinez 
I changed the German text; it took more than 30 mins to find a good sentence from Wikipedia. everything on the page are German, you know... a little bit time consuming. 😄

**Review by josejuanmartinez — APPROVED** [review]
Now it looks good. Thank you!

File: docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md (hunk 2 above)
Comment (josejuanmartinez) [reply docs/_posts/murat-gunay/2021-12-25-xlm_roberta_large_token_classifier_conll03_de.md]: @murat-gunay  we always need to do it. I showed the previous sentence to a German speaker and he was horrified :)

**Pull request merged**